Yoast/wordpress-seo
# Overhaul content types page

## Summary

This PR can be summarized in the following changelog entry:

* Overhauls the Content Types section under SEO -> Search Appearance by sectioning the post types and allowing users to collapse them. This is especially handy when you have a lot of custom post types.
* Bumps Yoast Components to 4.3.0.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch.
* Ensure that you've run `yarn install`, `grunt build:js` and `grunt build:css`.
* Navigate to SEO -> Search Appearance and note that the custom post type sections can be toggled.
* Navigate to Posts -> Add New and ensure the Snippet Editor looks the same way as it used to.

## Quality assurance
